### ISSUE TITLE
com.stripe.stripeterminal.external.models.Wallet is not Serializable

### ISSUE THREAD

**Issue by Ekalips** [stripe/stripe-terminal-android#408]
## Summary
It seems that `Stripe SDK version 3.2.0` started including a Wallet object into the PaymentIntent object which causes issue when app tries to serialize the PaymentIntent

## SDK version
3.2.0

## Crash log
```
Caused by java.io.NotSerializableException: com.stripe.stripeterminal.external.models.Wallet
       at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1240)
       at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1604)
       at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1565)
       at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1488)
       at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1234)
       at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1604)
       at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1565)
       at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1488)
       at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1234)
       at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1604)
       at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1565)
       at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1488)
       at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1234)
       at java.io.ObjectOutputStream.writeArray(ObjectOutputStream.java:1434)
       at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1230)
       at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1604)
       at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1565)
       at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1488)
       at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1234)
       at java.io.ObjectOutputStream.defaultWriteFields(ObjectOutputStream.java:1604)
       at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1565)
       at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1488)
       at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1234)
       at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:354)
       at android.os.Parcel.writeSerializable(Parcel.java:1709)
       at [REDACTED].ReaderPaymentResult.writeToParcel(ReaderPaymentResult.kt:22)
```

The ReaderPaymentResult class
```
@Parcelize
data class ReaderPaymentResult(
    val paymentIntentId: String,
    val paymentAmount: Double,
    val tipAmount: Double,
    val paymentIntent: PaymentIntent
): Parcelable
```

**Comment by chr-stripe** [comment]
Thank you for the report on this! We'll be releasing a fix soon that addresses this issue.

**Comment by chr-stripe** [comment]
This issue should now be resolved by SDK 3.2.1.
